src/main/java/th/ac/kmitl/science/comsci/example/models/Address.java

```diff
@@ -0,0 +1,16 @@
+package th.ac.kmitl.science.comsci.example.models;
+
+public class Address {
+    
+    private String Province;
```

Comment:
in Java variables naming convention, variables name should be writing in Lower Camal Case.
```java
Province -> province
Subdistrict -> subDistrict
PostCode -> postCode
Street -> street
Village -> village
```